src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -1297,6 +1306,94 @@ private BoundExpression RewriteDecimalConversion(BoundConversion oldNode, CSharp
             }
         }
 
+        private BoundExpression RewriteMethodGroupConversion(BoundConversion oldNode, CSharpSyntaxNode syntax, BoundExpression operand, MethodSymbol targetMethod, bool isExtensionMethod, TypeSymbol type)
```

Comment:
It looks like there is only one call site and isExtensionMethod is never true for it. Please remove the parameter.